ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -433,36 +413,6 @@ void startKsql(final KsqlConfig ksqlConfigWithPort) {
     initialize(ksqlConfigWithPort);
   }
 
-  void startApiServer(final KsqlConfig ksqlConfigWithPort) {
-    vertx = Vertx.vertx(
-        new VertxOptions().setMaxWorkerExecuteTimeUnit(TimeUnit.MILLISECONDS)
-            .setMaxWorkerExecuteTime(Long.MAX_VALUE));
-    vertx.exceptionHandler(t -> log.error("Unhandled exception in Vert.x", t));
-
-    final Endpoints endpoints = new KsqlServerEndpoints(
-        ksqlEngine,
-        ksqlConfigWithPort,
-        pullQueryExecutor,
-        ksqlSecurityContextProvider,
-        ksqlResource,
-        streamedQueryResource,
-        serverInfoResource,
-        heartbeatResource,
-        clusterStatusResource,
-        statusResource,
-        lagReportingResource,
-        healthCheckResource,
-        serverMetadataResource,
-        wsQueryEndpoint
-    );
-    apiServerConfig = new ApiServerConfig(ksqlConfigWithPort.originals());
-    apiServer = new Server(vertx, apiServerConfig, endpoints, true, securityExtension,
-        authenticationPlugin, serverState);
-    apiServer.setJettyPort(getJettyPort());
-    apiServer.start();
-    log.info("KSQL New API Server started");
```

Comment:
Can we add logging for various states of the server starting, similar to this? The old superclass from rest-utils did this for us but we've lost it as we no longer extend that class. I find these info messages extremely useful for debugging startup issues.

Reply:
Ack